fzf-tab.zsh

```diff
@@ -513,6 +530,25 @@ toggle-fzf-tab() {
     fi
 }
 
+fzf-tab-build-module() {
+  pushd $FZF_TAB_HOME/modules
+  CPPFLAGS=-I/usr/local/include CFLAGS="-g -Wall -O3" LDFLAGS=-L/usr/local/lib ./configure --disable-gdbm --without-tcsetpgrp
+  make -j
+  popd
+}
+
+if [[ -e $FZF_TAB_HOME/modules/Src/aloxaf/fzftab.so ]]; then
+  module_path+=("$FZF_TAB_HOME/modules/Src")
+  zmodload aloxaf/fzftab
```

Comment:
This fails for me:
```
$ echo $module_path
/usr/local/Cellar/zsh/5.8/lib /Users/zach.himsel/.zsh/zplug/repos/Aloxaf/fzf-tab/modules/Src

$ zmodload aloxaf/fzftab
zsh: failed to load module `aloxaf/fzftab': dlopen(/Users/zach.himsel/.zsh/zplug/repos/Aloxaf/fzf-tab/modules/Src/aloxaf/fzftab.bundle, 9): image not found
```

For reference, that was run on a Mac with homebrew's `zsh`.

Reply:
Umm, strange
Can you use zinit's zmodules?